Drivers/seds/TfLunaHwasComponent.xml

```diff
@@ -0,0 +1,390 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DataSheet xmlns="http://www.ccsds.org/schema/sois/seds" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.ccsds.org/schema/sois/seds seds.xsd">
+  <Device name="TfLunaComponent" />
+  <Package name="com/n7space/TfLunaHwasComponent">
+    <ComponentSet>
+      <Component name="TfLunaHwasDriver">
+        <LongDescription>SEDS TfLuna driver implementation</LongDescription>
+        <ProvidedInterfaceSet>
+          <Interface name="TfLunaHwas" type="com/n7space/TfLunaHwas/TfLunaHwasInterfaceType"></Interface>
+        </ProvidedInterfaceSet>
+        <RequiredInterfaceSet>
+          <Interface name="UartHwas" type="com/n7space/UartHwas/UartHwasInterfaceType" />
+          <Interface name="TfLunaReturnHwas" type="com/n7space/TfLunaHwas/TfLunaReturnHwasInterfaceType" />
+        </RequiredInterfaceSet>
+        <Implementation>
+          <VariableSet>
+            <Variable name="uartVar" type="com/n7space/UartHwas/UartHwas"></Variable>
+            <Variable name="uartConfigVar" type="com/n7space/UartHwas/UartHwasConfig" />
+            <Variable name="dataVar" type="com/n7space/TfLunaHwas/TfLunaHwas_Data"></Variable>
+            <Variable name="readByteVar" type="com/n7space/Hwas/Word"></Variable>
+            <Variable name="checksumVar" type="com/n7space/Hwas/Word"></Variable>
+            <Variable name="readStateVar" type="com/n7space/TfLunaHwas/TfLunaHwas_ReadSM"></Variable>
+          </VariableSet>
+          <ActivitySet>
+            <Activity name="InitTfLuna">
+              <Body>
+                <Assignment outputVariableRef="uartConfigVar.mInstance">
+                  <LongDescription>Set Uart instance</LongDescription>
+                  <Value value="UartHwas_Instance_2" />
+                </Assignment>
+                <Assignment outputVariableRef="uartConfigVar.mBaudrate">
+                  <LongDescription>Set Uart Baudrate</LongDescription>
+                  <Value value="UartHwas_Baudrate9600" />
+                </Assignment>
+                <SendCommandPrimitive shortDescription="Configure UART2" interface="UartHwas" command="InitUartCmd">
+                  <ArgumentValue name="uart">
+                    <VariableRef variableRef="uartVar" />
+                  </ArgumentValue>
+                  <ArgumentValue name="config">
+                    <VariableRef variableRef="uartConfigVar" />
+                  </ArgumentValue>
+                </SendCommandPrimitive>
+                <Assignment outputVariableRef="readStateVar">
+                  <LongDescription>Init state machine</LongDescription>
+                  <Value value="TfLunaHwas_ReadSM_NotFound" />

```

Comment:
Maybe this state could be called Idle, or Wait/Waiting? NotFound might be misleading

Reply:
Sure, I will change that

Reply:
Done